sidebars.js

```diff
@@ -352,7 +352,14 @@ module.exports = {
       label: 'Contributing',
       collapsed: true,
       items: [
-	    { type: 'doc', id: 'contributing/contributing' }
+	    { type: 'doc', id: 'contributing/contributing' },
+	    { type: 'doc', id: 'contributing/contributing-setup' },
+	    { type: 'doc', id: 'contributing/contributing-checklist' },
+	    { type: 'doc', id: 'contributing/contributing-github' },
+	    { type: 'doc', id: 'contributing/contributing-test' },
+	    { type: 'doc', id: 'contributing/contributing-maturity' },
+	    { type: 'doc', id: 'contributing/contributing-style' },
+	    { type: 'doc', id: 'contributing/contributing-misc' }
```

Comment:
I'm not sure why but these non-human IDs are showing up in the rendered TOC instead of the proper titles. Can you take a look here?

Reply:
The titles were missing